packages/radio-group/theme/lumo/vaadin-radio-group-styles.js

```diff
@@ -29,6 +29,12 @@ const radioGroup = css`
     padding-bottom: 0.7em;
   }
 
+  /* TODO: remove when the following components are updated to use new indicator:
+  combo-box, date-picker, time-picker, date-time-picker, select. */
```

Comment:
The only remaining component is date-time-picker, please update the comment accordingly.

Reply:
Done.